akka-docs-new/src/main/paradox/guide/actors-intro.md

```diff
@@ -0,0 +1,259 @@
+   Clients of such services need to be aware that tasks/messages might get lost during restarts. 
+   Even if loss does not happen, a response might be delayed arbitrarily due to previously enqueued tasks 
+   (a long queue), delays caused by garbage collection, etc. In face of these, concurrent systems should handle response
+   deadlines in the form of timeouts, just like networked/distributed systems.**  
+
+## How the actor model meets the needs of concurrent, distributed systems
+
+As described in the sections above, common programming practices cannot properly address the needs of modern concurrent
+and distributed systems. 
+Thankfully, we don’t need to scrap everything we know. Instead, the actor model addresses these shortcomings in a 
+principled way, allowing systems to behave in a way that better matches our mental model.
+
+In particular, we would like to:
+
+ * Enforce encapsulation without resorting to locks
+ * Use the model of cooperative entities reacting to signals, changing state and sending signals to each other 
+   to drive the whole application forward
+ * Stop worrying about the executing mechanism ("snaky-sneaky" threads) which is in a mismatch with our world view
+   	
+The actor model accomplishes all of these goals. The following topics describe how.
+   	
+### Use of message passing avoids locking and blocking
+  	
+Instead of calling methods, actors send messages to each other. Sending a message does not transfer the thread 
+of execution from the sender to the destination. An actor can send a message and continue without blocking. 
+It can do more work, send and receive messages.  	
+
+With objects, when a method returns, it releases control of its executing thread. In this respect, actors behave 
+much like objects, they react to messages and return execution when they finish processing the current message. 
+In this way, actors actually achieve the execution we imagined for objects:
+
+TODO: ACTOR-GRAPH
+
+An important consequence of passing messages instead of calling methods is that messages have no return value. 
+By sending a message, an actor delegates work to another actor. As we saw in [The illusion of a call stack], 
+if it expected a return value, the sending actor would either need to block (issue: locks and sacrifice of a thread), 
+or to execute the other actor’s work on the same thread (issue: "snakes-in-a-maze"). 
+Instead, the receiving actor delivers the results in a reply message.
+
+The second key change we need in our model is to reinstate encapsulation. Actors react to messages just like objects 
+"react" to methods invoked on them. The difference is that instead of multiple threads "protruding" into our actor and 
+wreaking havoc to internal state and invariants, actors execute independently from the senders of a message, and they 
+react to incoming messages sequentially, one at a time. There is always at most one message being processed, meaning 
```

Comment:
perhaps make it even more clear that it's "one at a time" per actor instance, so that it's not misinterpreted to "one at a time" for all actor instances